.github/workflows/BackendCD.yml

```diff
@@ -56,6 +59,9 @@ jobs:
         with:
           name: BuildOutput
           path: ./
+      - name: Add JWT secrets to Google Cloud 🔐
+        run: |
+          echo "  JWT_SECRET: ${{ secrets.JWT_SECRET }}" >> api.yaml
```

Comment:
Is that space at the beginning of the `echo` necessary? Just wanted to make sure since YAML is super picky with spaces and tabs.

Reply:
yeah the spaces are there intentionally because of exactly what you're talking about. The `api.yaml` file has a couple of environment variables so it needs to be indented a bit for yaml to see it as part of the list

Reply:
Perfect sounds good!